torch/csrc/utils/tensor_dtypes.cpp

```diff
@@ -12,6 +12,20 @@ namespace utils {
 
 std::pair<std::string, std::string> getDtypeNames(at::ScalarType scalarType) {
   switch (scalarType) {
+    case at::ScalarType::UInt1:
+      return std::make_pair("uint1", "");
```

Comment:
To accept the "bit" suggestion, change this to `"uint1", "bit"`

Reply:
OK. not sure how this works, do I use bit for all uint1 to uint7? and what does that mean?

Reply:
this looks like just an alias? do you mean to have bit1 to bit7?

Reply:
I think, it's to introduce a more discoverable alias for `uint1` = `bit1` :)